builders.py

```diff
@@ -450,16 +450,13 @@ def mklintyfactory():
 
 config_slaves = names(get_slaves(run_config=True, py27=True))
 
-sa0710 = 'sqlalchemy==0.7.10'
 sa087 = 'sqlalchemy==0.8.7'
```

Comment:
Technically speaking we doesn't test with 0.8.0, but state BB should work on it. However if anyone uses 0.8.* he probable uses version close to latest patch-level release of 0.8, so this should be ok.